### PR TITLE
Fix: docusaurus deployment configuration Closes #167

### LINKED ISSUE #167: Fix docusaurus deployment configuration
## Description

As a maintainer,
I need Docusaurus release action fixed,
So that documentation can be available on GitHub pages.

The deployment of Docusaurus to GitHub pages is failing.
Check out the error from deploying Docusaurus page to GH Pages: https://github.com/anitab-org/anitab-forms-web/runs/3771335051

Config file: https://github.com/anitab-org/anitab-forms-web/blob/develop/.github/workflows/docs.yml
Docusaurus config: https://github.com/anitab-org/anitab-forms-web/blob/develop/docs/docusaurus.config.js

### Update [Required]
- [ ] Fix the Docusaurus configuration file in order for the release to work
- [ ] Modify email from docs.yml to use `opensource@anitab.org` instead of `actions@github.com`

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation

1h

### PR DESCRIPTION
**Pull request by Aniket762**
### Description
Modified email from docs.yml to use `opensource@anitab.org` instead of `actions@github.com` and changed username from `gh-actions` to `anitab-org`. Moreover changed the project name as suggested by docusaurus docs

Fixes #167 

### Type of Change:

- Code
- Documentation

### Checklist:

- [X] My PR follows the style guidelines of this project
- [X] I have performed a self-review of my own code or materials
- [X] I have commented on my code or provided relevant documentation, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**
- [X] My changes generate no new warnings
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by isabelcosta — DISMISSED** [review]
@Aniket762 thank you for your contribution!
This looks correct to me :)

**Comment by isabelcosta** [comment]
I would not worry about the check fail, I feel its unrelated to the changes from this PR

**Review by codesankalp — DISMISSED** [review]

**Comment by codesankalp** [comment]
> I would not worry about the check fail, I feel its unrelated to the changes from this PR

Yes, it's unrelated. I will create an issue for this.

**Review by Aaishpra — DISMISSED** [review]
Lgtm🙌🏻

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ module.exports = {
   onBrokenMarkdownLinks: 'warn',
   favicon: 'img/favicon.ico',
   organizationName: 'AnitaB.org',
```
Comment (vj-codes) [inline docs/docusaurus.config.js]:
```suggestion
  organizationName: 'anitab-org',
```

**Review by vj-codes — CHANGES_REQUESTED** [review]
@Aniket762 hey congrats on your first pull request to the repo 🎉  I've requested one small change, after that it's good to approve :)

File: docs/docusaurus.config.js (hunk 1 above)
Comment (vj-codes) [reply docs/docusaurus.config.js]: Suggested because the organizationName should match with GitHub org name which at present differs

Comment (isabelcosta) [reply docs/docusaurus.config.js]: Nice catch @vj-codes 🙌🏾

**Review by isabelcosta — APPROVED** [review]

**Review by vj-codes — APPROVED** [review]
LGTM 🎉

**Comment by isabelcosta** [comment]
I will merge this, I think this will make the docusaurus work. I already have the deploy keys set up.

**Comment by isabelcosta** [comment]
Amazing!! it works 🥳 https://anitab-org.github.io/anitab-forms-web/

**Comment by Aniket762** [comment]
> Amazing!! it works 🥳 https://anitab-org.github.io/anitab-forms-web/

Thanks a lot, @isabelcosta for giving me this opportunity ✨ 

Looking forward to more such contributions and working with the team 🙈

**Pull request merged**